src/components/app.jsx

```diff
@@ -62,6 +63,44 @@ export const Application = () => {
     }, [dispatch]);
 
     useEffect(() => {
+        const channelList = cockpit.channel({
+            payload: "fslist1",
+            path: "/run/anaconda/",
+            superuser: "try",
+            watch: false,
+        });
+
+        channelList.addEventListener("message", (ev, data) => {
+            const file = JSON.parse(data);
+
+            if (file.path === "backend_ready") {
+                setBackendReady(true);
+            }
+        });
+
+        const channel = cockpit.channel({
+            payload: "fswatch1",
+            path: "/run/anaconda/",
+            superuser: "try",
+        });
+
+        channel.addEventListener("message", (ev, data) => {
+            const item = JSON.parse(data);
+
+            if (item.event === "created" && item.path === "/run/anaconda/backend_ready") {
```

Comment:
if I recall correctly you can use `cockpit.file("/run/anaconda/backend_ready").watch()` directly on a non-existing file.

Reply:
I was wondering that's why I asked you to review :P ok I will check!

Reply:
Works perfectly thanks